### ISSUE TITLE
Allow species in `yValues` of the visualization table

### ISSUE THREAD

**Issue by paulflang** [PEtab-dev/PEtab#569]
**Which problem would you like to address? Please describe.**
Atm it seems like [only observables can be plotted on the y axis](https://petab.readthedocs.io/en/stable/documentation_data_format.html#id4). It would be nice to also be able to plot species.

**Additional context**
This would be helpful to visually check
- if the behavior of non-observed species for a given optimization result seems biologically reasonable
- how species differ between experimental conditions
- how species differ between different optimization results, etc.

**Comment by dweindl** [comment]
Agreed, this would be quite helpful. When changing that, we should probably just allow any model entity with globally unique identifier (e.g., parameters, compartments).

**Comment by dweindl** [comment]
I think for the current way of plotting based on simulation files we could allow for pretty much everything that is present in that file. It would be an abuse of `observableId`, though. For practical purposes, I think it would already work if you add your species to a simulation file and let libpetab plot it. There is currently no validation for simulation files.
